lib/diagnostic.rb

```diff
@@ -93,13 +94,15 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array =
+arr.delete[2]
+arr.delete[3]
 
 # Taking the result from the question above, what would `arr` now equal:
 arr[arr.length + 2] = 99
 # Write your response as a ruby string
 
-Diagnostic.array_add_to = 'your answer here'
+Diagnostic.array_add_to = [12, 34, 56, 67, nil, 99]
 
```

Comment:
So close! Remember that arrays start at index 0 (`arr[0]` is the first element)
So it would be `[12, 34, 56, 67, nil, nil, 99]`